lib/matplotlib/rcsetup.py

```diff
@@ -301,6 +301,12 @@ def validate_color_or_auto(s):
     return validate_color(s)
 
 
+def validate_color_or_edge(s):
```

Comment:
Please keep private
```suggestion
def _validate_color_or_edge(s):
```

Reply:
Is changing the other ```validate_color_or_inherit``` function to a private one also recommended? Possibly in another PR?

Reply:
That would require a deprecation, so probably a new PR if it's done.